moped-editor/src/views/dev/LookupsView/settings.js

```diff
@@ -40,6 +42,20 @@ const workTypeHandler = (workTypes) =>
     <div key={workType.moped_work_type.id}>{workType.moped_work_type.name}</div>
   ));
```

Comment:
for example, should this be defined instead (expand to see line 37)
`const workTypeHandler = (workTypes, row) => `